compute/rn/release-information/release-notes-30-03-update3.adoc

```diff
@@ -0,0 +1,174 @@
+:toc: macro
+== 30.03 Update 3 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|30.03.122
+
+|Code name
+|Maxwell, 30.03 Update 3
+
+|Release date
+|July 16, 2023
+
+|Type
+|Maintenance Release
+
+|SHA-256
+|552f167c250e7bba0a1c7402951a9b240b269c88c7ac55a0c4ee109cdc5676c3
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/30/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtimes, tools, and orchestrators.
+
+//You can download the release image from the Palo Alto Networks Customer Support Portal, or use a program or script (such as curl, wget) to download the release image directly from our CDN: 
+//
+// LINK
+
+toc::[]
+
+//[#cve-coverage-update]
+//=== CVE Coverage Update
+
+[#enhancements]
+=== Enhancements
+//CWP-47358
+==== Package URL (pURL) Format Support for Vulnerabilities
+
+Prisma Cloud now adds package URLs (pURL) for packages and vulnerabilities while https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/tools/twistcli_scan_images[scanning the images using twistcli]. This helps to reliably reference the same software package using a simple and expressive syntax and conventions based on familiar URLs.
+
+As you scan images with `twistcli`, you can filter the CVEs based on the pURL format in the CVE viewer.
+
+image::cve-viewer-purl-filter.png[scale=60]
+
+//CWP-44490
+==== Registry Scan Scale Support Increased to 100k Images
+
+Registry image scan limit has been increased to 1,000,000 from 100,000 for all image types.
+
+//CWP-29369
+==== Support for GitLab Container Registry
+
+Added support for https://docs.paloaltonetworks.com/prisma/prisma-cloud/30/prisma-cloud-compute-edition-admin/vulnerability_management/registry_scanning/scan-gitlab[scanning GitLab Container Registry] without using admin credentials to manage and get a full list of all container registries/images.
+
+[#new-features-agentless-security]
+=== New Features in Agentless Security
+
+//CWP-44086
+==== Scan errors visibility improvements - accounts/ region error scope
```

Comment:
I suggest: 
==== Scan Errors Visibility Improvements in Agentless Scanning

Cc: @rcaballeromx